package questionV14_64;

//class MyException extends RuntimeException {
//
//}

//public class QE90 {
//
//	public static void main(String[] args) {
//		try {
//
//			method1(); // never will used because handles try catch block
//
//		} catch (MyException ne) {
//			System.out.println("A");
//		}
//
//	}
//
//	public static void method1() {
//		try {
//			System.out.println(Math.random());
//			
//			throw Math.random() > 0.5 ? new MyException() : new RuntimeException();
//
//		} catch (RuntimeException re) { // en kucuk exception basl
//
//			System.out.println("B");// B Always will be print 
//
//		}
//	}
//}
